fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/utils/infinispan/CacheLoaderChunkInputStream.java

```diff
@@ -194,6 +194,6 @@ private void endOfStream() {
         if (blobCache.contains(chunkKey)) {
             return blobCache.load(chunkKey).getValue();
         }
-        return null;
+        return new byte[]{};
```

Comment:
Returning `null` in this `private` method is probably cleaner and more efficient than Sonar's suggestion.